catalog/kyverno/releases.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kyverno
+  namespace: kyverno
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: kyverno
+      version: v2.6.5
+      sourceRef:
+        kind: HelmRepository
+        name: kyverno
+      interval: 1m
+  install:
+    crds: Create
+    remediation:
+      retries: 10
```

Comment:
idk if we need that, havent specified that anywhere. User should decide how to handle failures imo

Reply:
it removed.